pkg/combustion/combustion.go

```diff
@@ -50,3 +54,29 @@ func Configure(ctx *image.Context) error {
 
 	return nil
 }
+
+func generateComponentPath(ctx *image.Context, componentDir string) string {
+	return filepath.Join(ctx.ImageConfigDir, componentDir)
+}
+
+func isComponentConfigured(ctx *image.Context, componentDir string) bool {
```

Comment:
We can't have this in the build package because we'd run into circular dependency once again. Maybe a centralised "component" package makes sense for this? Probably in a follow up though.